src/Core/RulesAccessTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\rules\Core\RulesAccessTrait.
+ */
+
+namespace Drupal\rules\Core;
+
+use Drupal\Core\Access\AccessResult;
+use Drupal\Core\Session\AccountInterface;
+
+/**
+ * Implements access related functions for Rules plugins.
+ */
+trait RulesAccessTrait {
```

Comment:
sry I should have check usage first: it should  be "PluginAccessTrait" then + "PluginAccessInterface" imho